Plugins/GRPCProtobufGeneratorCommand/Plugin.swift

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import PackagePlugin
+
+@main
+struct GRPCProtobufGeneratorCommandPlugin: CommandPlugin {
+  /// Perform command, the entry-point when using a Package manifest.
```

Comment:
when using a package manifest? Not sure I understand this one.

Reply:
Ah, I forgot to implement the path which applies to Xcode projects `XcodeCommandPlugin`.

Reply:
Is that what it means? How do you invoke a command plugin via Xcode?

Reply:
Yes, I believe so. IIRC you right click on a package in the project navigator and any commands are in there.

Reply:
You don't get a choice of flavour: you can only invoke the Xcode flavour using that UI.